dlrover/python/unified/backend/elastic/worker/runner.py

```diff
@@ -45,9 +45,9 @@ def __init__(self, job_name: str, master: str, args: List[str]) -> None:
         self.master = master
         self.args = args
 
-        os.environ[
-            NodeEnv.DLROVER_MASTER_SERVICE_TYPE
-        ] = CommunicationType.COMM_SERVICE_RAY
+        os.environ[NodeEnv.DLROVER_MASTER_SERVICE_TYPE] = (
+            CommunicationType.COMM_SERVICE_RAY
+        )
         RayMasterClient.register_master_actor(master)
```

Comment:
need to rebase master after https://github.com/intelligent-machine-learning/dlrover/pull/1586